jsonnet/thanos-receive-controller-mixin/config.libsonnet

```diff
@@ -1,9 +1,10 @@
 {
   local thanos = self,
-  // Hierarcies is a way to help mixin users to add high level hierarcies to their alerts and dashboards.
+  // Hierarchies is a way to help mixin users to add high level hierarchies to their alerts and dashboards.
   // The key in the key-value pair will be used as label name in the alerts and variable name in the dashboards.
   // The value in the key-value pair will be used as a query to fetch available values for the given label name.
-  hierarcies+:: {
+  // old spelling for backwards compatibility
```

Comment:
Where do we use the old spelling?

Reply:
Ah, thanks! I tried to figure out a way to keep the old spelling for backwards compatibility but didn't find a good way in the end. This was a left-over comment from that attempt, removed it now.

Reply:
Got it. I think we can live with this breaking change, so should be fine 🙂.